aeron-driver/src/main/java/uk/co/real_logic/aeron/driver/MediaDriver.java

```diff
@@ -195,6 +209,7 @@ public static MediaDriver launch(final Context ctx)
     /**
      * Shutdown the media driver by stopping all threads and freeing resources.
      */
+    @Override
```

Comment:
What is the value in adding "@Override"? It just clutters up the code and is poorly defined for Java as most uses are implements rather than a true override.